instrumentation/github.com/gin-gonic/gin/otelgin/gintrace.go

```diff
@@ -75,7 +75,7 @@ func Middleware(service string, opts ...Option) gin.HandlerFunc {
 			opts = append(opts, oteltrace.WithAttributes(rAttr))
 		}
 		ctx, span := tracer.Start(ctx, spanName, opts...)
-		defer span.End()
+		defer span.End(cfg.SpanEndOptions...)
```

Comment:
I didn't write a test for this, as the only option is `WithStackTrace`, and I could only trigger it with a `panic()`, which wouldn't really allow a test.